modules/ingestor/src/graph/organization.rs

```diff
@@ -83,8 +83,14 @@ impl Graph {
                 let mut entity = organization::ActiveModel::from(found.organization);
                 entity.website = Set(information.website);
                 entity.cpe_key = Set(information.cpe_key);
-                let model = entity.update(&self.connection(&tx)).await?;
-                Ok(OrganizationContext::new(found.graph, model))
+                let model = match entity.update(&self.connection(&tx)).await {
+                    Ok(model) => Ok(model),
+                    Err(e) => {
+                        log::error!("Failed to update organization: {e}");
```

Comment:
Unless this terminates or seriously affects the the application, this should be reduced to info or debug.